drivers/docker/driver.go

```diff
@@ -739,6 +739,20 @@ func (d *Driver) createContainerConfig(task *drivers.TaskConfig, driverConfig *T
 		config.WorkingDir = driverConfig.WorkDir
 	}
 
+	containerRuntime := driverConfig.Runtime
+	if _, ok := task.DeviceEnv[nvidiaVisibleDevices]; ok {
+		if !d.gpuRuntime {
+			return c, fmt.Errorf("requested docker-runtime %q was not found", d.config.GPURuntimeName)
```

Comment:
```suggestion
			return c, fmt.Errorf("requested docker runtime %q was not found", d.config.GPURuntimeName)
```

having the dash there makes it sounds like `docker-runtime` is it's own configuration value rather than just `runtime`.